helix-view/src/editor.rs

```diff
@@ -149,6 +153,14 @@ pub struct Config {
         deserialize_with = "deserialize_duration_millis"
     )]
     pub idle_timeout: Duration,
+    /// Time in milliseconds since last keypress before a redraws trigger.
+    /// Used for redrawing asynchronously computed UI components, set to 0 for instant.
+    /// Defaults to 100ms.
```

Comment:
Says it defaults to `100` but the code uses 200. We should set this to `DIFF_DEBOUNCE_TIME_ASYNC` maybe? Or double that.

I also think this shouldn't be configurable by end users because it's easy to misconfigure (same as `idle-timeout` which really shouldn't be `0` but it's set to `0` by a lot of users). If complaints come in we can always make it smarter or configurable at that point.

Reply:
That makes a lot of sense. I was trying to follow the precedent of the `idle-timeout`.
I hard-coded this to 96ms (`DIFF_DEBOUNCE_TIME_ASYNC`)

Reply:
Sorry for hijacking with something unrelated to this PR. I want to mention that the current documentation explicitly states “set to 0 for instant” so if 0 is a bad idea that section could be altered to explain why 0 ought to be avoided and what’d be recommended instead